### PR TITLE
Bugfix: #201 could not decorate gitlab merge request

### PR DESCRIPTION
**Pull request by PiekJ**
Enables GitLab merge request to be decorated. This PR fixes issue #201 for SonarQube 8.2+.

### CODE REVIEW

File: src/test/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/SetBindingActionTest.java [hunk 1]
```diff
@@ -21,9 +21,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (mc1arke) [inline src/test/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/SetBindingActionTest.java:24]: Please don't use star imports

Comment (PiekJ) [reply src/test/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/SetBindingActionTest.java]: The change has been reverted.

**Comment by PiekJ** [comment]
I resolved your comment about the star import and squashed my commits.

File: src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/gitlab/SetGitlabBindingAction.java [hunk 2]
```diff
@@ -27,15 +28,25 @@
...
 
     public SetGitlabBindingAction(DbClient dbClient, ComponentFinder componentFinder, UserSession userSession) {
         super(dbClient, componentFinder, userSession, "set_gitlab_binding");
     }
 
+    @Override
+    protected void configureAction(WebService.NewAction action) {
+        super.configureAction(action);
+        action.createParam(REPOSITORY_PARAMETER).setRequired(true);
+    }
```
Comment (mc1arke) [inline src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/gitlab/SetGitlabBindingAction.java:41]: Please add a unit test for this new method

File: src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/gitlab/SetGitlabBindingAction.java [hunk 3]
```diff
@@ -27,15 +28,25 @@
 import com.github.mc1arke.sonarqube.plugin.server.pullrequest.ws.action.SetBindingAction;
 
 public class SetGitlabBindingAction extends SetBindingAction {
+    private final static String REPOSITORY_PARAMETER = "repository";
```
Comment (mc1arke) [inline src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/gitlab/SetGitlabBindingAction.java:31]:
```suggestion
    private static final String REPOSITORY_PARAMETER = "repository";
```

**Comment by PiekJ** [comment]
I resolved all the suggestions, last week. Is it ready to merge?

**Comment by PiekJ** [comment]
I resolved the suggestions.

**Comment by mc1arke** [comment]
Merged. Thanks for the contribution!

**Pull request merged**